### PR TITLE
ADD Post FontAwesome and Post react init

### PR DESCRIPTION
**Pull request by cristofer-dev**
- Add post how add FontAwesome in ReactAPP
- Add post react init general react app

 Changes to be committed:
	new file:   src/assets/img/fontawesome.svg
	new file:   src/assets/img/react.svg
	renamed:    src/posts/2019-08-24-tools.md -> src/posts/2018-08-24-tools.md
	renamed:    src/posts/2020-09-08-autorizador-apigateway.md -> src/posts/2018-09-08-autorizador-apigateway.md
	new file:   src/posts/2018-09-23-fontawesome.md
	new file:   src/posts/2018-09-27-react-init.md

### CODE REVIEW

File: src/posts/2018-08-24-tools.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ Integrando eslint, airbnb rules y prettier en nuestro flujo de trabajo.
...
 ### Quien, que y por que.
 
-- **eslint**: un linter para JavaScript.
-- **airbnb style**: guía de estilo para JavaScript. ( usadas por el lint )
-- **prettier**: Es un formateador de código, al que le diremos que use las reglas de eslint para aplicar formato al código de forma automatica.
-- **husky**: Nos permite definir acciones que seran ejecutas antes de hacer un `commit` evaluando ciertas reglas que definamos, en este caso, forzaremos a ejecucción de prettier que formateara nuestro código aplicando las reglas de airbnb.
-- **lint-staged**: Corre `eslint` a los archivos que estan en `stage` antes del commit.
+* **eslint**: un linter para JavaScript.
+* **airbnb style**: guía de estilo para JavaScript. ( usadas por el lint )
+* **prettier**: Es un formateador de código, al que le diremos que use las reglas de eslint para aplicar formato al código de forma automatica.
```
Comment (Xabadu) [inline src/posts/2018-08-24-tools.md]: `automática`

File: src/posts/2018-08-24-tools.md [hunk 2]
```diff
@@ -9,11 +9,11 @@ Integrando eslint, airbnb rules y prettier en nuestro flujo de trabajo.
...
+* **husky**: Nos permite definir acciones que seran ejecutas antes de hacer un `commit` evaluando ciertas reglas que definamos, en este caso, forzaremos a ejecucción de prettier que formateara nuestro código aplicando las reglas de airbnb.
```
Comment (Xabadu) [inline src/posts/2018-08-24-tools.md]: `serán ejecutadas`
`ejecución`
`formateará`

File: src/posts/2018-08-24-tools.md [hunk 3]
```diff
@@ -55,14 +55,41 @@ Le decimos a eslint que reglas aplicar y con quien debe integrarse, creando `.es
...
+  "plugins": ["prettier"],
+
+  "rules": {
+    "prettier/prettier": ["error"],
+    "camelcase": "off"
+  }
 }
 ```
 
+Tambien podemos sobre-escribir usando prettier definiendo reglas en el archivo `.prettierrc` en la raiz del proyecto.
```
Comment (Xabadu) [inline src/posts/2018-08-24-tools.md]: `También`
`raíz`

File: src/posts/2018-09-27-react-init.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+[mini-css-extract-plugin](https://webpack.js.org/plugins/mini-css-extract-plugin/)
+[babel-loader](https://webpack.js.org/loaders/babel-loader/)
+[css-loader](https://webpack.js.org/loaders/css-loader/)
+[file-loader](https://webpack.js.org/loaders/file-loader/)
+
+## Extras
+
+- i18next
+- Linter
+- fontAwwesome
```
Comment (Xabadu) [inline src/posts/2018-09-27-react-init.md]: `Awesome`

File: src/posts/2018-09-27-react-init.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+- Graficos
```
Comment (Xabadu) [inline src/posts/2018-09-27-react-init.md]: `Gráficos`

**Review by Xabadu — CHANGES_REQUESTED** [review]

**Comment by cristofer-dev** [comment]
Done @Xabadu ! :)
Thanks!

File: src/posts/2018-08-24-tools.md [hunk 6]
```diff
@@ -11,8 +11,8 @@ Integrando eslint, airbnb rules y prettier en nuestro flujo de trabajo.
 
 * **eslint**: un linter para JavaScript.
 * **airbnb style**: guía de estilo para JavaScript. ( usadas por el lint )
-* **prettier**: Es un formateador de código, al que le diremos que use las reglas de eslint para aplicar formato al código de forma automatica.
-* **husky**: Nos permite definir acciones que seran ejecutas antes de hacer un `commit` evaluando ciertas reglas que definamos, en este caso, forzaremos a ejecucción de prettier que formateara nuestro código aplicando las reglas de airbnb.
+* **prettier**: Es un formateador de código, al que le diremos que use las reglas de eslint para aplicar formato al código de forma automática.
+* **husky**: Nos permite definir acciones que serán ejecutas antes de hacer un `commit` evaluando ciertas reglas que definamos, en este caso, forzaremos a ejecucción de prettier que formateará nuestro código aplicando las reglas de airbnb.
```
Comment (Xabadu) [inline src/posts/2018-08-24-tools.md]: `ejecutadas`

**Review by Xabadu — CHANGES_REQUESTED** [review]

**Review by Xabadu — APPROVED** [review]

**Comment by cristofer-dev** [comment]
Thanks @Xabadu !

**Pull request merged**